tools/lsp/Cargo.toml

```diff
@@ -109,5 +109,8 @@ wasm-bindgen-futures = "0.4.30"
 [dev-dependencies]
 spin_on = "0.1"
 
+[build-dependencies]
+slint-build = { workspace = true, features = ["default"] }
```

Comment:
should be optional and depend on the preview feature.

If this is just about splitting in different files, you could also have `import`'ed from the `slint!`  macro.  (But i'm not objecting to use slint-build. Whatever you prefer)